### PR TITLE
created Portfolio skeleton from figma template 

### PR DESCRIPTION
**Pull request by hellohimesh**
- created project with linter and github workflow
- converted figma templates HTML CSS mobile skeleton 
- applied custom fonts to heading and description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <a><img src="assets/ic_menu.png" alt="menu icon"></a>
+            </div>
+        </div>
+
+    </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Please make sure the container of the section is a header tag as stated in the requirement👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <a><img src="assets/ic_menu.png" alt="menu icon"></a>
+            </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure the `menu icon` is wrapped in a `nav` tag👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        <div class="collaborate_button">
+            start collaboration
+        </div>
+
+    </div>
+    <div class="container">
+
+        <img class="image_geometry_1" src="assets/image_geometry_2-1.png" alt="image_geometry_2-1">
+
+    </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure the container of this section has a `section` tag to make your page more semantic😎

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </div>
+        <div class="collaborate_button">
+            start collaboration
+        </div>
+
+    </div>
+    <div class="container">
+
+        <img class="image_geometry_1" src="assets/image_geometry_2-1.png" alt="image_geometry_2-1">
```
Comment (rotimiazeez) [inline index.html]: - [x] Please note that all section decorations are background images, not `IMG` tag👌 kindly fix that🙏

File: index.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                <a><img src="assets/ic_menu.png" alt="menu icon"></a>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="container">
+        <div class="title_name">
+            Himesh Suthar
+        </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Please make the main title a `h1` tag and not a div for a [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)page😎

File: index.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        <div class="about_description">
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly note that this is a paragraph and should be wrapped in a `p` tag.😊

File: index.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        <div class="collaborate_button">
+            start collaboration
+        </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure the button follow the button styling of the Figma design
- Your design
screenshot (28)
-  Figma design
screenshot (30)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+    <div class="project_container">
+        <div class="project_title_holder">
+            Project
+        </div>
+
+        <div class="container">
+            <img class="Rectangle_image" src="assets/Rectangle-21.png" alt="Rectangle_image" />
+
+        </div>
```
Comment (rotimiazeez) [inline index.html]: - [x] Please make sure your design match with the original template design👏 
- Your design
screenshot (31)
- Figma design
screenshot (29)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @himesh-suthar 😊, 

### Good job so far!👏
### To Highlight!👍
- Good README ✔
- Descriptive commit messages✔
- No linter error✔
#### There are some issues that you still need to work on to go to the next project but you are almost there!
> Just a few changes to be made to make you unstoppable💯🚀
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (hellohimesh) [reply index.html]: screenshot
 Done

File: index.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        </p>
+        <a><div class="collaborate_button">
+            <p id="start-collaboration" >start collaboration</p>
+        </div>
+        </a>
+
+    </section>
+    <div class="image-container">
+
+        <!-- <img class="image_geometry_1" src="assets/image_geometry_2-1.png" alt="image_geometry_2-1"> -->
```
Comment (brytebee) [inline index.html]: - [x] **Required** Let's remove all commented out code. It's bad practice to commit commented out code and this makes your code clean.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+
+    <div class="project_container">
+        <div class="project_title_holder">
+            Project
+        </div>
+
+        
+            <!-- <img class="Rectangle_image" src="assets/Rectangle-21.png" alt="Rectangle_image" /> -->
+            <!-- <div style="background-image: url('assets/Rectangle-21.png');"></div> -->
```
Comment (brytebee) [inline index.html]: - [x] **Required** Let's remove all commented out code. As stated above

File: index.html [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+    <div class="image-container">
+
+        <!-- <img class="image_geometry_1" src="assets/image_geometry_2-1.png" alt="image_geometry_2-1"> -->
+        <div id ="image_geometry_2-1"></div>
+    </div>
+
+    <div class="project_container">
+        <div class="project_title_holder">
+            Project
+        </div>
```
Comment (brytebee) [inline index.html]: - [x] **Optional** Consider making all your containers [HTML semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) such as `<section>`, `<article>`, etc

File: style1.css [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+.ic_menu {
+  position: absolute;
+  width: 20px;
+  height: 14px;
+  right: 15px;
+  top: 19px;
+
+  /* background: #1A2236; */
```
Comment (brytebee) [inline style1.css]: - [x] **Required** Let's remove all commented out code. As stated above

File: style1.css [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+
+.flex-container {
+  display: flex;
+  align-items: center;
+  position: absolute;
+
+  /* width: 288px;
+  height: 336px;
+  left: 16px; */
+}
```
Comment (brytebee) [inline style1.css]: - [x]  **Required** Let's remove all commented out code. As stated above

File: style1.css [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+  height: 100px;
+  left: 16px;
+  top: 104px;
+  margin-top: 0;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 3.2em;
+  line-height: 100%;
+
+  /* or 50px */
```
Comment (brytebee) [inline style1.css]: - [x]  **Required** Let's remove all commented out code. As stated above

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi @himesh-suthar 👋,

Great work on making the changes.👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <nav><img src="assets/ic_menu.png" alt="menu icon"></nav>
```
Comment (ShoiraTa) [inline index.html:22]: - [ ] [OPTIONAL]: Please consider putting `hamburger` menu inside a `<button>` tag
- [ ] [OPTIONAL]: Please check the `hamburger` menu, it changed it's location and immigrated to the left side far away 😄 

image

File: index.html [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+        </h1>
+        <p class="about_description">
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <a><div class="collaborate_button">
+            <p id="start-collaboration" >start collaboration</p>
+        </div>
+        </a>
```
Comment (ShoiraTa) [inline index.html:41]: - [ ] [OPTIONAL]: `start collaboration` is a `button` 🔘 , so it should be:
```
<button>
  start collaboration
</button>
```

File: index.html [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+
+    </section>
+    <section class="image-container">
+        <div id ="image_geometry_2-1"></div>
+    </section>
+
+    <section class="project_container">
+        <div class="project_title_holder">
+            Project
```
Comment (ShoiraTa) [inline index.html]: - [ ]  [OPTIONAL]: I recommend you to put text  inside relevant tags, like `<p><h1><h2><h3>.. `

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi STUDENT,  

Your project is complete! Your page looks awesome 🤗 . Congratulations!  🎉💯🌟 

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Professional documentation ✔️ 



## Optional suggestions

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- [ ] I strongly recommend to work on changes suggested below 🌞 

***
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 17 above)
Comment (hellohimesh) [reply index.html]: I already push this changes

File: index.html (hunk 16 above)
Comment (hellohimesh) [reply index.html:41]: Yes, I will do that. I thought to use <a> which can be used to call function from js and have added on hover method in CSS to look like button

File: index.html (hunk 15 above)
Comment (hellohimesh) [reply index.html:22]: thanks for the suggestions. I appreciate it

**Pull request merged**